components/List/index.js

```diff
@@ -0,0 +1,48 @@
+import {
+  listItemStyles,
+  checkboxStyles,
+  textStyles,
+  cardStyles,
+  buttonStyles,
+} from "@/styles";
+
+export default function List({ todos, onToggleCheckTodo, onDeleteTodo }) {
+  return (
+    <>
+      {todos.length > 0 && (
+        <div style={cardStyles}>
+          <ul>
+            {todos
+              .slice()
+              .reverse()
+              .map((todo) => {
+                return (
+                  <li key={todo.id} style={listItemStyles}>
+                    <input
+                      type="checkbox"
+                      checked={todo.isChecked}
+                      onChange={() => {
+                        onToggleCheckTodo(todo.id);
+                      }}
+                      style={checkboxStyles}
```

Comment:
never thought of importing the styles like you did, wish i had seen this earlier 🥲 Good work though!!